ChatBot/models/dialogue/domain.json

```diff
@@ -0,0 +1,48 @@
+{
+  "states": [
+    "intent_Female",
+    "intent_Male",
+    "intent_affirm",
+    "intent_botPlace",
+    "intent_deny",
+    "intent_gender",
+    "intent_goodbye",
+    "intent_greet",
+    "intent_identity",
+    "intent_informAge",
+    "intent_informThird",
+    "intent_intro",
+    "intent_medicines",
+    "intent_prevent",
+    "intent_safeMedicine",
+    "intent_thankyou",
+    "entity_medicine",
+    "entity_age",
+    "slot_age_0",
+    "slot_matches_0",
+    "slot_medicine_0",
+    "slot_requested_slot_0",
+    "prev_action_listen",
+    "prev_action_restart",
+    "prev_action_default_fallback",
+    "prev_utter_default",
+    "prev_utter_greet",
+    "prev_utter_goodbye",
+    "prev_utter_youarewelcome",
+    "prev_utter_identity",
+    "prev_utter_place",
+    "prev_utter_prevent",
+    "prev_utter_intro",
+    "prev_utter_medicines",
+    "prev_utter_ask_name",
+    "prev_utter_ask_gender",
+    "prev_utter_ask_pregnant",
+    "prev_action_patient_information",
+    "prev_action_suggest_pregnant",
+    "prev_action_suggest_normal",
+    "prev_utter_ask_age",
+    "prev_check_age",
+    "prev_action_check_medicine_normal",
+    "prev_action_check_medicine_pregnant"
+  ]
+}
```

Comment:
are these autogenerated.. if not add a new line.

Reply:
This is an autogenerated model file.